doc/administrator/editing.rst

```diff
@@ -57,6 +57,13 @@ a string of the form ``[<schemaname>.]<tablename>``.  If ``schemaname`` is
 omitted, the table is assumed to be in the ``public`` schema.  The label
 corresponding to this field is *Related Postgres table* in the admin interface.
 
+The ``geoTable`` field support named formating and can get values from the
+ environment variables, for example:
+
+ ``[<schemaname>_{some-named-variable}.]<tablename>``
+
+with "some-named-variable" defined in the environment.
```

Comment:
```suggestion
 ``[<database-connexion-name>.[<schemaname>_{some-named-variable}.]]<tablename>``
```

It should probably not added here, but it's missing...

Reply:
I have never seen this case ever before, is it an actual existing possibility ? 
do you have a practical example of that ?

Reply:
As I remember @asaunier use it recently, it is linked to the ``dbsessions``, see: https://camptocamp.github.io/c2cgeoportal/master/integrator/multiple_databases.html

Reply:
ok thanks

Reply:
Thanks :-)